packages/core/content-type-builder/admin/src/components/ListRow/index.js

```diff
@@ -1,22 +1,27 @@
 import React, { memo } from 'react';
 import PropTypes from 'prop-types';
 import get from 'lodash/get';
+import styled from 'styled-components';
 import { useIntl } from 'react-intl';
 import { IconButton } from '@strapi/design-system/IconButton';
 import { Flex } from '@strapi/design-system/Flex';
 import { Stack } from '@strapi/design-system/Stack';
 import { Typography } from '@strapi/design-system/Typography';
+import Lock from '@strapi/icons/Lock';
 import Pencil from '@strapi/icons/Pencil';
 import Trash from '@strapi/icons/Trash';
 import { stopPropagation, onRowClick } from '@strapi/helper-plugin';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import useDataManager from '../../hooks/useDataManager';
 import getTrad from '../../utils/getTrad';
 import Curve from '../../icons/Curve';
 import UpperFist from '../UpperFirst';
 import BoxWrapper from './BoxWrapper';
 import AttributeIcon from '../AttributeIcon';
 
+const ActionsPlaceholder = styled.div`
```

Comment:
Could we use this instead?
`import { pxToRem } from '@strapi/helper-plugin'`
then
`<Box height={pxToRem(32)}>`

Reply:
Tbh I would say to use the DS every time it is possible to avoid custom - since we had lots of custom code before the v4+DS. I'm more toward why using `div` if we can use `Box`  🤷🏻‍♀️
Following @HichamELBSI learnings 🤓🙏

Reply:
+1 I think we need to reuse our existing components. Since we can do this with the Box component and it would save us from adding custom code to the codebase.

Reply:
Thanks for the feedback from the both of you. I think I get the idea 👍🏽